conformance/tests/generics_paramspec_semantics.py

```diff
@@ -0,0 +1,143 @@
+"""
+Tests semantics of ParamSpec.
+"""
+
+# Specification: https://typing.readthedocs.io/en/latest/spec/generics.html#semantics
+
+
+from typing import Callable, Concatenate, Generic, ParamSpec, TypeVar, assert_type
+
+P = ParamSpec("P")
+
+
+def changes_return_type_to_str(x: Callable[P, int]) -> Callable[P, str]:
+    ...
+
+
+def returns_int(a: str, b: bool, /) -> int:
```

Comment:
It's a little confusing that use of positional-only means this example doesn't match the spec. Should we remove that / change the spec / just add a comment?

Reply:
The exact signature isn't important for purposes of this test. I used position-only parameters so I could use an `assert_type` with a `Callable` type to verify the results.